### ISSUE TITLE
Validate the latest cert-manager in Rancher server helm chart to latest

### ISSUE THREAD

**Issue by bashofmann** [rancher/rancher#39728]
Currently we recommend to install cert-manager 1.7.1 (https://docs.ranchermanager.rancher.io/pages-for-subheaders/install-upgrade-on-a-kubernetes-cluster#4-install-cert-manager). cert-manager 1.7.1 is EOL; requesting eng validation for cert-manager in Rancher to see if it can be updated to 1.10.x (or higher). This will require an update to docs as well. See https://cert-manager.io/docs/installation/supported-releases/ for currently supported releases.

See also https://github.com/rancher/rancher/issues/36693 as a reference to the previous validation testing done on 1.7.1.

SURE-5786

**Comment by thecrazyrussian** [comment]
Just to add a comment to this, with Rancher being validated to k8s 1.24 but cert manager 1.7.x only to 1.23. This effectively means that Rancher is only deploy-able to k8s 1.23 with cert-manager.

Rancher Support Matrix: https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-7-0/
Up to k8s 1.24
Upgrade Docs Recommending 1.7: https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/resources/upgrade-cert-manager
Cert Manager K8s support: https://cert-manager.io/docs/installation/supported-releases/
Where 1.7 is only supported up to 1.23

**Comment by holtakj** [comment]
same issue here. would like to know to which version to upgrade.

**Comment by thecrazyrussian** [comment]
> same issue here. would like to know to which version to upgrade.

Not the official word here, but we pushed to 1.8 and tested reissuing, which worked for LetsEncrypt. We did not test renewal that may or may not be effected by lack of `rotationPolicy` in the Rancher/LetsEncrypt generated certs. Not sure about how the validity check works for letsencrypt, it may not yet be tied to rotationPolicy.

1.7.3 also worked charmingly on k3s 1.24 but again we have not tested renewal.
